### DISCUSSION TITLE
npm create equivalent

### DISCUSSION THREAD

**Discussion by o-az** [jsr-io/jsr#594] [Q&A]
is there currently a command equivalent to `npm create`?

**Comment by Thesephi** [comment]
I'm aware of `deno init` and `deno init --lib` ([new in 1.45](https://deno.com/blog/v1.45#deno-init---lib)) but I also look forward to having the `npm create <some-arbitrary-template>` (e.g. `npm create cloudflare`) equivalence in Deno. So +1 on this thread.
